src/components/LandingPage/LandingPage.css

```diff
@@ -0,0 +1,101 @@
+#landing-page-container {
+    display: flex;
+    justify-content: center;
+    max-width: 100%;
+}
+
+#info-section {
+    display: flex;
+    flex-direction: column;
+    max-width: 40%;
+    flex: 1;
+    margin-top: 5%;
+    margin-right: 3%;
+}
+
+#title {
+    font-weight: 700px;
+    color: #000000;
+    font-size: 54px;
+    font-style: normal;
+    letter-spacing: 0.035em;
+    line-height: 56px;
+    display: flex;
+    text-align: left;
+}
+
+#lab-description {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 24px;
+    letter-spacing: 0.04em;
+    display: flex;
+    text-align: left;
+    margin-top: 15px;
+}
+
+.btn_ubc_home {
+    display: flex;
+    width: 219px;
+    height: 42px;
+    background-color: #77B0F3;
+    font-weight: 700;
+    letter-spacing: 0.04em;
+    font-size: 15px;
+    line-height: 19px;
+    color: #ffffff;
+    text-align: center;
+    padding-bottom: 6px;
+    justify-content: center;
+    margin-top: 20px;
+}
+
+#landing-page-img-container {
+    margin-top: 75px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    overflow: hidden;
+    max-width: 50%;
+}
+
+#landing-page-img-placeholder {
+    background: #C4C4C4;
+    padding: 10px;
+    width: 491px;
+    height: 296px;
+    flex: 1;
+}
+
+@media (max-width:1025px){
+    #landing-page-container {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+
+    #info-section {
+        margin: auto;
+        justify-content: center;
+        max-width: 70%;
+    }
+
+    #title {
+        text-align: center;
+    }
+
+    .btn_ubc_home {
+        margin-left: 30px;
+    }
```

Comment:
It seems that this specific `margin-left` is not compatible with some of the screens in terms of responsiveness, as I sometimes get this:

<img width="400" alt="Screen Shot 2022-11-04 at 1 58 25 PM" src="https://user-images.githubusercontent.com/71746168/200074065-ecad823f-b16c-4582-8b17-0873a564349e.png">


What do you think of this approach instead? It seems really responsive on my end:

```suggestion
    .btn_ubc_home {
        margin-left: auto;
        margin-right: auto
    }
```